### ISSUE TITLE
Use distance from user (when known) as a tiebreaker for linguistic matches

### ISSUE THREAD

**Issue by riordan** [pelias/api#207]
# tl;dr
If we know a user's location or area of interest (inferred from viewport) AND a search results in a tie for linguistic matches, then we should use the distance from the user as a tiebreaker.

# A Starbucks is a Starbucks is a Starbucks
<img src="https://cloud.githubusercontent.com/assets/227472/9468914/4c2a5568-4b12-11e5-9d5b-ada6999a1d71.png" width="300px" alt="screenshot of Open by Mapzen app searching for a Starbucks while located in Manhattan. The top ranked result, while a Starbucks, is many miles away from the closest Starbucks to the user">

Today, when a search result returns a linguistic match, but has the end-user's location to bias the result, it returns results within "the bubble" around the user. However, if results are tied for matches (i.e. Starbucks hires a social media team to standardize their data in OSM), we select the order of results based on the import order into the system.

Ideally, when the location data is provided by the end-user or end-user's device, we should use that to rank these matching results.

# [A rose is a rose is a rose.](https://en.wikipedia.org/wiki/Rose_is_a_rose_is_a_rose_is_a_rose)

Except for when one rose is closer to you.

**Comment by riordan** [comment]
NOTE TO SELF: we should write something up in the wiki about our guidelines for handling location bias between lat-lon and extent/Viewport. This can form the basis of our developer suggestions about providing location bias to pelias.

**Comment by riordan** [comment]
We're doing this.
